src/screenComponents/customShipFunctions.h

```diff
@@ -3,11 +3,13 @@
 
 #include "playerInfo.h"
 #include "gui/gui2_autolayout.h"
+#include "screenComponents/radarView.h"
 
 class GuiCustomShipFunctions : public GuiAutoLayout
 {
 public:
     GuiCustomShipFunctions(GuiContainer* owner, ECrewPosition position, string id);
+    GuiCustomShipFunctions(GuiContainer* owner, ECrewPosition position, string id, TargetsContainer* targets);
```

Comment:
Instead of 2 constructors we can just have 1 with a default:
```GuiCustomShipFunctions(GuiContainer* owner, ECrewPosition position, string id, TargetsContainer* targets=nullptr);```